CHANGES

```diff
@@ -1,960 +0,0 @@
-Apache CouchDB CHANGES
```

Comment:
This is not a CHANGELOG for `erlang-pbkdf2` as can be seen from the contents of the file.